lib/simctl.js

```diff
@@ -6,8 +6,7 @@ import _ from 'lodash';
 
 const log = getLogger('simctl');
 
-async function simExec (command:string, timeout:number, args:Array = [],
-    env = {}) {
+async function simCommand (command:string, timeout:number, args:Array = [], env = {}, executingFunction) {
```

Comment:
(to get defaults like this, I think you can make a call like `simCommand(undefined, undefined, undefined, arg)`)